docs/ossm/versioning-and-upgrades/README.md

```diff
@@ -70,9 +70,13 @@ The strategy to use when updating the Istio control plane. The available strateg
 ### Istio control plane update strategies
 The `Istio` resource configuration includes the `spec.updateStrategy` field, which determines how the Istio control plane will be updated. When the operator observes a change in the Istio `spec.version` field or if a new minor release becomes available and a `vX.Y-latest` version alias is configured, an upgrade procedure will be initiated. The available strategies are `InPlace` and `RevisionBased`.
 
-When the `InPlace` strategy is used, the existing Istio control plane is updated in-place. The workloads therefore don't need to be moved from one control plane instance to another, but take into account that the applications proxy will need to be updated as well, In-Place strategy is not recommended as a strategy to be used in production environments because it can cause traffic disruption, to minimize the disruption, ensure that at least two replicas of istiod are running. When the `RevisionBased` strategy is used, a new Istio control plane instance is created for every change to the `.spec.version` field. The old control plane remains in place until all workloads have been moved to the new control plane instance.
+When the `InPlace` strategy is used, the existing Istio control plane is updated and restarted in-place. This means that there is only one instance of the Istio control plane present during the upgrade process and workloads therefore do not need to be moved to the updated control plane instance. The update process is completed by restarting application workloads and gateways to update the Envoy proxies.
 
-The `RevisionBased` strategy is the recommended strategy for production environments, as it minimizes traffic disruption during the upgrade process. The `InPlace` strategy is recommended for non-production environments, where traffic disruption is acceptable.
+While the `InPlace` strategy is the simpler and more efficient update strategy, there is a small possibility of application traffic interruption in the event that a workload pod is updates, restarts or scales while the Istio control plane is restarting. This can be mitigated by running multiple replicas of the Istio control plane (Istiod).
```

Comment:
```suggestion
While the `InPlace` strategy is the simpler and more efficient update strategy, there is a small possibility of application traffic interruption in the event that a workload pod updates, restarts or scales while the Istio control plane is restarting. This can be mitigated by running multiple replicas of the Istio control plane (Istiod).
```